another_one.txt

```diff
@@ -1,2 +1,3 @@
 here we go

```

Comment:
Cool, but what if we did x or y instead here?

Reply:
a